scripts/webpack.config.js

```diff
@@ -116,9 +109,6 @@ function getConfig(isReactExternalized) {
             new DefinePlugin({
                 __LANGUAGE__: JSON.stringify(language),
                 __VERSION__: JSON.stringify(version),
-                __TOKEN__: JSON.stringify(token), // used for examples only
```

Comment:
The functionality of passing in token and item ID was only necessary for examples. In storybook, you can manually replace these arguments in the UI.